pkg/eraser/eraser.go

```diff
@@ -26,10 +26,11 @@ import (
 )
 
 var (
-	runtimePtr    = flag.String("runtime", "containerd", "container runtime")
-	imageListPtr  = flag.String("imagelist", "", "name of ImageList")
-	enableProfile = flag.Bool("enable-pprof", false, "enable pprof profiling")
-	profilePort   = flag.Int("pprof-port", 6060, "port for pprof profiling. defaulted to 6060 if unspecified")
+	runtimePtr      = flag.String("runtime", "containerd", "container runtime")
+	isCustomRuntime = flag.Bool("custom-runtime", false, "assume runtime passed is local socket path")
```

Comment:
One other gotcha is that the configuration system [recently moved from cli args to configmaps](https://github.com/Azure/eraser/pull/581). That means that the end-user no longer has a way to set an arg on an eraser container directly, and it would need to be handled via the configmap and propagated down to the container (through this arg, or through an env var).